react/features/mobile/permissions/functions.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+
+import { Alert, Linking, NativeModules } from 'react-native';
+
+import { Platform } from '../../base/react';
+
+/**
+ * Opens the settings panel for the current platform.
+ *
+ * @private
+ * @returns {void}
+ */
+export function openSettings() {
+    switch (Platform.OS) {
+    case 'android':
+        NativeModules.AndroidSettings.open().catch(() => {
+            Alert.alert(
+                'Error opening settings',
+                'Please open settings and grant the required permissions',
+                [
+                    { text: 'OK' }
+                ]
+            );
+        });
+        break;
+
+    case 'ios':
+        Linking.openURL('app-settings:');
+        break;
+    }
+}
```

Comment:
What is the settings panel and why is it in the permissions features? I ask because I have to open the web settings side-panel for the new toolbar and I'm looking for code I can re-use. (I already implemented on a local branch by, I think, creating a side-panel feature and having action that open the side panel.)

Reply:
It's the operating system's Settings for the installed app. You know where you can see the permissions the app has been granted and/or denied, whether it's allowed to do background refresh, whether it can use cellular data. It's nothing specific to jitsi-meet, it's not the in-app Settings panel on Web or the in-app Settings screen on mobile.